package.json

```diff
@@ -82,7 +83,8 @@
     "babel-eslint": "^10.1.0",
     "babel-jest": "^26.3.0",
     "babel-loader": "^8.1.0",
-    "commitizen": "^4.1.2",
+    "commitizen": "^4.2.1",
```

Comment:
`npm audit fix` updated this

Reply:
Thats weird that it rolled back, but ok

Reply:
it's not a rollback, it went from `^4.1.2` to `^4.2.1`